pages/faqs.md

```diff
@@ -26,16 +28,44 @@ We've launched the new version of Exercism, 18 months in the making :D
 
 There is a document that [explains the changes](/about-v1-to-v2). We hope you enjoy the new experience :)
 
+### How do new language tracks get added to the site?
+
+A new language track gets created when a member of the community takes the lead on it and becomes a maintainer of the track. If you'd like to get involved in helping set one up, there are [instructions here](https://github.com/exercism/request-new-language-track/blob/master/README.md).
+
+## Exercism CLI FAQs
+
 ### I can't submit!
 
 The new site has a brand new CLI, and also needs some extra metadata for your exercise. Read about how to [upgrade your CLI and migrate your solutions for the new site](/cli-v1-to-v2).
 
 If that doesn't help, or you didn't use the old version of the CLI, please read through the [CLI Walkthrough](https://exercism.io/cli-walkthrough). If that doesn't help either, open an issue on [exercism/exercism.io](https://github.com/exercism/exercism.io) and we will help you get it sorted out.
 
-### How do new language tracks get added to the site?
+### How to check if a new version of the CLI is available?
+The version command `exercism version` show the running version of the Exercism  CLI. By providing the latest command flag you check if there is a newer version of the CLI available `exercism version -l`
 
-A new language track gets created when a member of the community takes the lead on it and becomes a maintainer of the track. If you'd like to get involved in helping set one up, there are [instructions here](https://github.com/exercism/request-new-language-track/blob/master/README.md).
+### How do to upgrade to the latest version of the CLI?
+  The exercism CLI has an upgrade command that can be used to upgrade your CLI in-place (full replacement of existing binary). The command `exercism upgrade` will [check if a new version is available] and upgrade to the latest available version of the CLI.
+
+### Permission denied errors when upgrading!
+If you receive a permission error when trying to upgrade the CLI, chances are the binary has been installed into a directory that you don’t have permission writing to.
+
+To resolve this issue check the path of the Exercism CLI `which exercism` on Linux and MacOs. If the returned directory is not your home path please ensure that you have the write permissions to the path where the exercism binary exists. If you installed the binary using sudo then, you should try to upgrade by running `sudo exercism upgrade`.

```

Comment:
On windows we can use `where exercism`.

Also instead of blindly suggesting `sudo`, we should encourage removal of the old version and reinstall via one of the official ways.

Also especially for MacOS we should keep in mind that the `brew` installation is an official way and should not be upgraded by force, but by brew instead.

To be honest, I'd not suggest `sudo exercism upgrade` at all, if it has already been installed via a systems package manager a forced update might confuse the package manager.

And people using their systems package manager despite beeing discouraged from it **should knwo what they are doing** anyway.

If they used a package manager and its not updating fast enough, they have to ask the maintainer of that package or even better, re-install user based.